cli/ops.rs

```diff
@@ -229,7 +229,7 @@ fn op_now(
   let inner = msg::NowRes::create(
     builder,
     &msg::NowResArgs {
-      seconds,
+      millis,
```

Comment:
No. The op was good before.

The ts side needs to convert to millis.

Please revert this commit.

Reply:
already reverted :)